idea-factore/idea-factory
# First major milestone for contracts.

Implemented a lot of big features.
Tested on local blockchain
Fixed some deploy issues with hardhat


We are now able to:

* add ideas
*  create a pool
*  create a subpool
*  add an idea to a pool
*  add an idea to a subpool


Next big design stuff to think about should be whether a parent pool should also have any childPool ideas
and also collateral in a parent pool vs collateral in a child poo and how the two relate if at all
